compose-v2/galaxy-server/files/start.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+create_user() {
+  GALAXY_PROXY_PREFIX=$(cat $GALAXY_CONFIG_DIR/GALAXY_PROXY_PREFIX.txt)
+  echo "Waiting for Galaxy..."
+  until [ "$(curl -s -o /dev/null -w '%{http_code}' ${GALAXY_URL:-nginx}$GALAXY_PROXY_PREFIX)" -eq "200" ] && echo Galaxy started; do
```

Comment:
If you put ephemeris into the container, which I think is useful and we shiped it in the old one as well, you can use galaxy-wait https://ephemeris.readthedocs.io/en/latest/commands/galaxy-wait.html

Reply:
Hm, I'm not sure about adding ephemeris just for one single command 🤔 Are there other benefits right now? We could still add it in the future if the need arises, but this one-line `until` does the job quite well currently in my opinion.

Reply:
ephemeris has some other nice commands like the tool installation and tool testing, but you are right we can evaluate this later and can keep the container small for now